packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -29,6 +30,11 @@ def test_if_docker_app_can_be_deployed(cluster):
     cluster.marathon.deploy_test_app_and_check(*get_test_app_in_docker(ip_per_container=False))
 
 
+@pytest.mark.xfail(
+    dcos.config['security'] == 'strict',
```

Comment:
Please never use dcostests in anything new. I have duplicated all functionality of that code into the harness in this repo.